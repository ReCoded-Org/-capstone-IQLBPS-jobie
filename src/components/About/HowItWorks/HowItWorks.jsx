import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
import {  useTranslation } from "react-i18next";
import {
  faFileLines,
  faHandshake,
  faPen,
} from "@fortawesome/free-solid-svg-icons";

function HowItWorks() {
  const {t} = useTranslation();
  return (
    <div className="container px-20 ">
      <div>
        <h1 className="text-left text-5xl font-inter text-secondary py-10 font-bold">
          {t('how-it-works')}
        </h1>
      </div>
      <div className="text-left flex justify-center flex-nowrap content-center text- font-inter gap-20 small:flex-col small:items-center medium:flex-col pb-9">
        <div className=" ">
          <div className="rounded-full bg-dark-gray h-16 w-16 flex items-center justify-center ">
            <FontAwesomeIcon
              icon={faFileLines}
              className="text-3xl text-accent"
            />
          </div>
          <h1 className="text-3xl py-3 small:text-left">{t('create-your-resume')} </h1>
          <p className="text-justify small:text-justify small:text-xl">
            
            {t('creating-resume-details')}
          </p>
        </div>
        <div className=" ">
          <div className="rounded-full bg-dark-gray h-16 w-16 flex items-center justify-center">
            <FontAwesomeIcon icon={faPen} className="text-3xl text-accent" />
          </div>
          <h1 className="text-3xl py-3 small:text-left">{t('apply-for-jobs')}</h1>
          <p className="text-justify small:text-justify small:text-xl">
           {t('job-application-details')} 
          </p>
        </div>
        <div className=" ">
          <div className="rounded-full bg-secondary h-16 w-16 flex items-center justify-center">
            <FontAwesomeIcon
              icon={faHandshake}
              className="text-3xl text-accent"
            />
          </div>
          <h1 className="text-3xl py-3">{t('get-hired')}</h1>
          <p className="text-justify small:text-justify small:text-xl">
          {t('getting-hired')}
           
          </p>
        </div>
      </div>
    </div>
  );
}
export default HowItWorks;
