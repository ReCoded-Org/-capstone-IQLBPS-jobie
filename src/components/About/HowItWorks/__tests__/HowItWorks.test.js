import React from "react";
import renderer from "react-test-renderer";
import HowItWorks from "../HowItWorks";
it("renders Custom button correctly", () => {
  const tree = renderer.create(<HowItWorks />).toJSON();
  expect(tree).toMatchSnapshot();
});
