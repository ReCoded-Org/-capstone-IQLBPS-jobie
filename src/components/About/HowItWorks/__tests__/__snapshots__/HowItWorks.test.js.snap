// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Custom button correctly 1`] = `
<div
  className="container px-20 "
>
  <div>
    <h1
      className="text-left text-5xl font-inter text-secondary py-10 font-bold"
    >
      How it Works
    </h1>
  </div>
  <div
    className="text-left flex justify-center flex-nowrap content-center text- font-inter gap-20 small:flex-col small:items-center medium:flex-col pb-9"
  >
    <div
      className=" "
    >
      <div
        className="rounded-full bg-dark-gray h-16 w-16 flex items-center justify-center "
      >
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-file-lines text-3xl text-accent"
          data-icon="file-lines"
          data-prefix="fas"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 384 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M64 0C28.7 0 0 28.7 0 64V448c0 35.3 28.7 64 64 64H320c35.3 0 64-28.7 64-64V160H256c-17.7 0-32-14.3-32-32V0H64zM256 0V128H384L256 0zM112 256H272c8.8 0 16 7.2 16 16s-7.2 16-16 16H112c-8.8 0-16-7.2-16-16s7.2-16 16-16zm0 64H272c8.8 0 16 7.2 16 16s-7.2 16-16 16H112c-8.8 0-16-7.2-16-16s7.2-16 16-16zm0 64H272c8.8 0 16 7.2 16 16s-7.2 16-16 16H112c-8.8 0-16-7.2-16-16s7.2-16 16-16z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
      </div>
      <h1
        className="text-3xl py-3 small:text-left"
      >
        Create your resume 
      </h1>
      <p
        className="text-justify small:text-justify small:text-xl"
      >
        Create your resume and add your skills and experience to your profile. You can also add your education and work experience to your profile.
      </p>
    </div>
    <div
      className=" "
    >
      <div
        className="rounded-full bg-dark-gray h-16 w-16 flex items-center justify-center"
      >
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-pen text-3xl text-accent"
          data-icon="pen"
          data-prefix="fas"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M362.7 19.3L314.3 67.7 444.3 197.7l48.4-48.4c25-25 25-65.5 0-90.5L453.3 19.3c-25-25-65.5-25-90.5 0zm-71 71L58.6 323.5c-10.4 10.4-18 23.3-22.2 37.4L1 481.2C-1.5 489.7 .8 498.8 7 505s15.3 8.5 23.7 6.1l120.3-35.4c14.1-4.2 27-11.8 37.4-22.2L421.7 220.3 291.7 90.3z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
      </div>
      <h1
        className="text-3xl py-3 small:text-left"
      >
        Apply for jobs
      </h1>
      <p
        className="text-justify small:text-justify small:text-xl"
      >
        Search for jobs and apply for them. You can also search for companires and apply for their jobs.
      </p>
    </div>
    <div
      className=" "
    >
      <div
        className="rounded-full bg-secondary h-16 w-16 flex items-center justify-center"
      >
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-handshake text-3xl text-accent"
          data-icon="handshake"
          data-prefix="fas"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 640 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M323.4 85.2l-96.8 78.4c-16.1 13-19.2 36.4-7 53.1c12.9 17.8 38 21.3 55.3 7.8l99.3-77.2c7-5.4 17-4.2 22.5 2.8s4.2 17-2.8 22.5l-20.9 16.2L512 316.8V128h-.7l-3.9-2.5L434.8 79c-15.3-9.8-33.2-15-51.4-15c-21.8 0-43 7.5-60 21.2zm22.8 124.4l-51.7 40.2C263 274.4 217.3 268 193.7 235.6c-22.2-30.5-16.6-73.1 12.7-96.8l83.2-67.3c-11.6-4.9-24.1-7.4-36.8-7.4C234 64 215.7 69.6 200 80l-72 48V352h28.2l91.4 83.4c19.6 17.9 49.9 16.5 67.8-3.1c5.5-6.1 9.2-13.2 11.1-20.6l17 15.6c19.5 17.9 49.9 16.6 67.8-2.9c4.5-4.9 7.8-10.6 9.9-16.5c19.4 13 45.8 10.3 62.1-7.5c17.9-19.5 16.6-49.9-2.9-67.8l-134.2-123zM96 128H0V352c0 17.7 14.3 32 32 32H64c17.7 0 32-14.3 32-32V128zM48 352c-8.8 0-16-7.2-16-16s7.2-16 16-16s16 7.2 16 16s-7.2 16-16 16zM544 128V352c0 17.7 14.3 32 32 32h32c17.7 0 32-14.3 32-32V128H544zm64 208c0 8.8-7.2 16-16 16s-16-7.2-16-16s7.2-16 16-16s16 7.2 16 16z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
      </div>
      <h1
        className="text-3xl py-3"
      >
        Get Hired
      </h1>
      <p
        className="text-justify small:text-justify small:text-xl"
      >
        Once you have applied for a job you will be notified when the employer has responded to your application.
      </p>
    </div>
  </div>
</div>
`;
